src/screens/SiteDetailScreen.js

```diff
@@ -318,10 +341,10 @@ class SiteDetailScreen extends Component {
                             <View style={[bottomMargin, locationMainContainerStyle]}>
                                 <View>
                                     <Text style={textStyle}>
-                                        {campsite_form.latitude}: {coordinate.latitude}
+                                        {campsite_form.latitude}: {coordinate.latitude.toFixed(5)}
                                     </Text>
                                     <Text style={[textStyle, bottomMargin]}>
-                                        {campsite_form.longitude}: {coordinate.longitude}
+                                        {campsite_form.longitude}: {coordinate.longitude.toFixed(5)}
```

Comment:
GH-86 -- the long GPS coords were pushing other things off the screen. Now all of them are 5 decimals.